pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+log_cli = true
+log_cli_level = INFO
+log_level = NOTSET
+log_format = %(asctime)s %(levelname)s %(name)s %(funcName)s: %(message)s
```

Comment:
This change also impacts the dev process so let's please factor it into a separate PR in order to keep a clean focus. Thanks!